### PR TITLE
make the container easily configurable

### PR DESCRIPTION
**Pull request by TomasTomecek**
TODO

- [ ] should the changes be added to 9.4 and 9.2?

For more info, see https://github.com/TomasTomecek/flexible-postgresql-demo

### CODE REVIEW

**Comment by pkubatrh** [comment]
[test]

**Comment by TomasTomecek** [comment]
I would love to get some feedback here -- what maintainers think about this approach.

File: 9.5/root/usr/share/container-scripts/postgresql/README.md [hunk 1]
```diff
@@ -61,6 +61,29 @@ or if it was already present, [`postgres`](http://www.postgresql.org/docs/9.2/st
 is executed and will run as PID 1. You can stop the detached container by running
 `docker stop postgresql_database`.
 
+Supplying custom configuration file
```
Comment (omron93) [inline 9.5/root/usr/share/container-scripts/postgresql/README.md]: What about add notice about "how to supply custom configuration file"? This section is mainly about getting configuration template.

Comment (TomasTomecek) [inline 9.5/root/usr/share/container-scripts/postgresql/README.md]: good point! there's nothing about running postgres while using the custom template config; will add

Comment (TomasTomecek) [inline 9.5/root/usr/share/container-scripts/postgresql/README.md]: @omron93 I added a commit which should address your comment.

**Comment by pkubatrh** [comment]
Since we are now looking into the possibility of introducing the source-to-image functionality for daemon based images it might make sense to not merge the PR right now, as the way the configuration files are used might get changed (e.g. being stored in `/opt/app-root/src/' instead).

**Comment by TomasTomecek** [comment]
What would be the timeline for s2i-based postgresql?

**Comment by hhorak** [comment]
> What would be the timeline for s2i-based postgresql?

I'd say weeks, we already know how to do it (follow the ready-to-merge PR for mongodb available at https://github.com/sclorg/mongodb-container/pull/239).

**Comment by TomasTomecek** [comment]
closing this PR as per above

**Pull request closed without merging**